CREATE-NEW-SECRET-SOURCE.md

```diff
@@ -0,0 +1,83 @@
+# Create a new secret source
```

Comment:
This is what they call perfect documentation 💯 

It's limited to the essentials and yet super clear 🔝 🎩

Reply:
Thanks man! 😳